src/mame/layout/connect4.lay

```diff
@@ -3,47 +3,127 @@
 license:CC0
 -->
 <mamelayout version="2">
-	<element name="digit" defstate="0">
-	<led7seg>
-			<color red="1.0" green="0.0" blue="0.0" />
-	</led7seg>
+
+	<element name="ledseg0">
+		<image statemask="0x01">
+		<data>
+			<![CDATA[
+				<svg xmlns="http://www.w3.org/2000/svg" width="40" height="60" viewBox="0 0 109.2832 165.8748">
+				<path fill="#ffffff" stroke="#ffffff" stroke-opacity="1" fill-opacity="1" stroke-width="0.265px" d="M 22.20542,9.1048101 28.73216,15.40339 90.121429,15.68542 98.067028,7.8827048 97.972444,6.3785695 91.634871,-0.01402362 H 30.245613 L 22.489191,7.8827048 Z" />
```

Comment:
Do you actually need to stroke these paths?  They’re simple filled shapes – you’d be better off dropping the stroking and just adjusting the path so it follows the outlline of that you want filled.